src/IdentityManager/DataService/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace IdentityManager.Lib
```

Comment:
Remove unused class

Reply:
this is not our part, so as we merged to our branch from master, this class came from identity, and when they fix this we can merge again